api.yaml

```diff
@@ -879,13 +879,21 @@ components:
             $ref: "#/components/schemas/CloudAssetDetails"
     CloudAssets:
```

Comment:
This is a fix for a bug where generated swagger bindings could not parse response from Asset-Inventory due to mismatch between the code and api.yaml. This changes api.yaml to match code, as existing clients clearly are able to parse responses, so we are using code as source of truth.